app/main.py

```diff
@@ -1,5 +1,67 @@
+from __future__ import annotations
+
+
 class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self,
+                 name: str,
+                 weight: int | float,
+                 coords: list | float | None = None) -> None:
```

Comment:
`coords` can be float. Also, specify an inner type of list
```suggestion
                 coords: list[int] | None = None) -> None:
```